src/poprox_recommender/model/nrms/news_encoder.py

```diff
@@ -14,11 +15,11 @@ def __init__(self, config):
         self.config = config
 
         self.plm = AutoModel.from_pretrained(
-            self.config.pretrained_model, cache_dir="/tmp/"
```

Comment:
If I understand correctly this PR bakes most (all?) of this stuff into the container, but I'd still like to keep the `cache_dir='/tmp'` as an insurance policy because the issue that solves won't bite us until we try to run in an actual deployed environment

Reply:
Yep, the PR bakes this all into the container, but I'm happy to leave the cache dir there.

Reply:
@karlhigley Pushed a change that re-adds it, but using `tempfile.gettempdir()` instead of hard-coding `/tmp` — sound good?